### PR TITLE
Add styling to bookstore

### PR DESCRIPTION
**Pull request by Newtayo**
## Features

- [ ] Style book store to match zeplin design

### CODE REVIEW

File: src/components/Styles/Bookcard.css [hunk 1]
```diff
@@ -5,8 +5,16 @@
...
   padding: 2rem 6rem;
+  margin: 1.5rem auto 1rem auto;
+  background-color: #fff;
+  width: 90%;
+  border-radius: 2px;
+}
+
+.bookdetails {
+  margin: 0;
+  padding: 0;
```
Comment (AnselemOdims) [inline src/components/Styles/Bookcard.css:19]: - [ ] Good job working on the styling, but there little discrepancies that need to be taken care of. Please kindly compare the images below, you would notice that the font-family, font-weight, font-size and color of most of the text is different. Please kindly fix this.

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Newtayo , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Bookcard.jsx [hunk 2]
```diff
@@ -24,10 +24,13 @@ const Bookcard = ({ book }) => {
...
         </nav>
       </div>
-      <div className="progress">
-        <h3 className="percentage">{book.percent}</h3>
-        <p className="complete">Completed</p>
+      <div className="container">
+        <div className="oval" />
+        <div className="progress">
+          <h3 className="percentage">70%</h3>
+          <p className="complete">Completed</p>
```
Comment (OyePriscilla) [inline src/components/Bookcard.jsx:31]: - [ ] One more thing, kindly ensure to give this element as pointed out earlier the correct `font-family` (Montserrat) according to the given template. 👍 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Newtayo 👏 👏 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Add styling to app ✔️
Made some required changes ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Newtayo** [comment]
@OyePriscilla  The font family is already set to 'Montserrat' sans-serif' as it is in the design. please kindly clarify what you want me to correct

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Newtayo 

Your project is complete! Great job 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- zeplin design well followed
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by OyePriscilla** [comment]
> @OyePriscilla The font family is already set to 'Montserrat' sans-serif' as it is in the design. please kindly clarify what you want me to correct

@Newtayo No, it wasn't. That's the reason why I attached the screenshot for clarification.

**Pull request merged**